pkgs/applications/science/misc/root/default.nix

```diff
@@ -90,15 +88,14 @@ stdenv.mkDerivation rec {
     lapack
     libxcrypt
     libxml2
-    llvm_13
+    llvm_16
     lz4
     xz
     gsl
     gtest
     openblas
     openssl
     xxHash
-    libAfterImage
```

Comment:
Why is that? https://github.com/root-project/root/tree/master/graf2d/asimage/src/libAfterImage is still in the tree.

Reply:
Good question! Root is now forcing to statically link against builtin lol afterimage because it contains custom patches. I was not with my laptop today so I have a hard time finding the commit that made this change, but I'll upgrade on this tomorrow

Reply:
If libAfterImage is not using anymore, we need to remove the package (and alias needs to be added to `pkgs/top-level/aliases.nix`).

Reply:
Ok! Here is the relevant ROOT PR, for the record: https://github.com/root-project/root/pull/14395